lib/flutter_map_tappable_polyline.dart

```diff
@@ -120,7 +120,12 @@ class TappablePolylineLayer extends StatelessWidget {
 
         return Container(
           child: GestureDetector(
+              onDoubleTap: () {
```

Comment:
doing this we can double-tap on the map and zoom. Is this a solution to #7 ?

Reply:
It seems to be! Nicely done. 👏

Reply:
It should do the job